client/js/reactComponents.jsx

```diff
@@ -132,8 +132,8 @@ class ChatMessage extends React.Component {
 	render() {
 		var username = dataManager.users[this.props.data.sender].username;
 		return (
-			<div style={{"backgroundColor": "rgba(255, 255, 255, 0.5)", "marginTop":"2px"}}>
-				<span style={{"fontSize": "70%", "color":"white"}}>{username}<br/></span>
```

Comment:
The span with the username is a separate element from the enclosing div, as it is now both the message and the username has the same style as declared in .username, so it doesn't look the same. You could do something like

.message {
 background-color = rgba...
 ...
}

.message>span {
 color: white;
 ....
}

You could also create a separate class for the message and for the username, but that might be overkill for this temporary solution.
@Frankrike